README.md

```diff
@@ -1,16 +1,34 @@
-# Sway Applications
-
-## Introduction
+<p align="center">
+    <img src="./logo.png" height="120">
+</p>
+
+<p align="center">
+    <a href="https://github.com/FuelLabs/sway-applications/actions/workflows/ci.yml" alt="CI">
+        <img src="https://github.com/FuelLabs/sway-applications/actions/workflows/ci.yml/badge.svg" />
+    </a>
+    <a href="https://crates.io/crates/forc/0.13.2" alt="forc">
+        <img src="https://img.shields.io/badge/forc-v0.13.2-orange" />
+    </a>
+    <a href="https://discord.gg/xfpK4Pe">
+        <img src="https://img.shields.io/discord/732892373507375164?color=blue&logo=discord&logoColor=ffffff&labelColor=6A7EC2&label=Discord" />
+    </a>
+</p>
```

Comment:
Avoid raw HTML. Use markdown for including images. See https://github.com/FuelLabs/fuels-ts/blob/master/README.md?plain=1 for reference.

Reply:
I did initially use markdown however that does not center align the badges and logo. It looks better that way. What is the reason for not using HTML?

Reply:
Ah I see. Can you just wrap the markdown including the badges and image in a `<p align="center"> ... </p>` block, or does that not work?

Reply:
I will try it and get back to you.

Reply:
Nope, markdown does not support positioning. [Ref](https://stackoverflow.com/a/12091563)

We can either use HTML to align or we do not position in the center

Reply:
Up to you, don't feel strongly about either direction.